sdk/client.go

```diff
@@ -17,10 +17,10 @@ import (
 type Client interface {
 	Checker(ctx context.Context, check *health.CheckState) error
 	Health() *healthcheck.Client
-	PostJob(ctx context.Context, headers Headers) (models.Job, error)
-	PatchJob(ctx context.Context, headers Headers, jobID string, body []PatchOperation) (RespHeaders, error)
-	PostTasksCount(ctx context.Context, headers Headers, jobID string, payload []byte) (RespHeaders, models.Task, error)
-	GetTask(ctx context.Context, headers Headers, jobID, taskName string) (RespHeaders, *models.Task, error)
+	PostJob(ctx context.Context, headers Headers) (*models.Job, error)
+	PatchJob(ctx context.Context, headers Headers, jobID string, body []PatchOperation) (*RespHeaders, error)
+	PostTasksCount(ctx context.Context, headers Headers, jobID string, payload []byte) (*RespHeaders, *models.Task, error)
```

Comment:
this is already implemented, merge the changes from develop to see this implementation